mypy_django_plugin/main.py

```diff
@@ -270,8 +270,12 @@ def get_attribute_hook(self, fullname: str) -> Optional[Callable[[AttributeConte
             return partial(settings.get_type_of_settings_attribute, django_context=self.django_context)
 
         info = self._get_typeinfo_or_none(class_name)
+        if info and info.has_base(fullnames.PERMISSION_MIXIN_CLASS_FULLNAME) and attr_name == "is_superuser":
+            return partial(set_auth_user_model_boolean_fields, django_context=self.django_context)
         if info and info.has_base(fullnames.HTTPREQUEST_CLASS_FULLNAME) and attr_name == "user":
             return partial(request.set_auth_user_model_as_type_for_request_user, django_context=self.django_context)
+        if info and info.has_base(fullnames.ABSTRACT_USER_MODEL_FULLNAME) and attr_name in ("is_staff", "is_active"):
```

Comment:
I am a big fan of class-level immutable constants for cases like this. Do you think that this is worth refactoring?

Reply:
`frozenset` basically 🙂

Reply:
do you mean moving `ABSTRACT_USER_MODEL_FULLNAME` to the `NewSemanalDjangoPlugin` class?

Reply:
No, I was thinking about `('is_staff', 'is_active')`

Reply:
ahhh
yeah, tbh i was trying to think of a way to turn it into a dict, so you could just pass in the fullname and attr_name and get the correct partial as a result, but i'm not sure that's doable in a clean way :/

i don't think that one bit is worth refactoring, but i absolutely want to refactor those three if statements

Reply:
Ok 👍